src/test/java/hudson/plugins/git/GitChangeSetTest.java

```diff
@@ -150,24 +164,24 @@ public void testFindOrCreateUser() {
     public void testIsoDate() {
 
         GitChangeSet c = new GitChangeSet(Arrays.asList("author John Doe <john.doe@jenkins-ci.org> 2015-03-03T09:22:42-0700"), true);
-        assertEquals("2015-03-03T09:22:42-0700",c.getDate());
+        assertEquals("2015-03-03T09:22:42-0700", c.getDate());
```

Comment:
:+1: I agree with Kostya. For good or for bad, white space changes for this plugin are not desirable. If someday in the future we decide to rationalize white space throughout the files, we would probably use an automated formatter as part of every build. Until then, we will try to keep white space changes to a minimum.